mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS com/mojang/blaze3d/platform/GlStateManager
+	CLASS com/mojang/blaze3d/platform/GlStateManager$DestFactor DestFactor
+	CLASS com/mojang/blaze3d/platform/GlStateManager$SourceFactor SourceFactor
```

Comment:
?

Reply:
Matcher and Enigma disagree on inner class ordering :thinking:

Reply:
looks like he just sorted them, they were way below previously

Reply:
This is fine. Don't worry